dev/set_matrix.py

```diff
@@ -380,6 +448,11 @@ def expand_config(config):
             "torch",
         )
         for category, cfg in cfgs.items():
+            if category not in VALID_CATEGORIES:
```

Comment:
This is follow-up for https://github.com/mlflow/mlflow/pull/12779#discussion_r1694833529